src/vs/workbench/common/editor.ts

```diff
@@ -602,6 +612,10 @@ export abstract class EditorInput extends Disposable implements IEditorInput {
 		return { typeId: this.typeId };
 	}
 
+	async requiresTrust(): Promise<boolean> {
```

Comment:
Please also provide a default implementation for the `SideBySideEditor` below which should return `true` if any of the two editors inside require trust.